fix(docs): smtp docs (#663)
scaleway/docs-content: compute/elastic-metal/how-to/enable-smtp.mdx

<Message type="tip"> 

> This guide explains how to enable SMTP on Elastic Metal servers. If you want to enable SMTP on an Instance, follow our documentation on [how to send emails from your Instance](/compute/instances/how-to/send-emails-from-your-instance) and [how to enable SMTP on Public Gateways](/network/vpc/how-to/configure-a-public-gateway/#how-to-enable-smtp)..

Reviewer: ```suggestion
This guide explains how to enable SMTP on Elastic Metal servers. If you want to enable SMTP on other resources, follow our documentation on [how to send emails from your Instance](/compute/instances/how-to/send-emails-from-your-instance) or [how to enable SMTP on Public Gateways](/network/vpc/how-to/configure-a-public-gateway/#how-to-enable-smtp).
```
